tests/unit/lms/services/user_test.py

```diff
@@ -118,37 +118,23 @@ def test_get_users(
             admin_organization_ids=[organization.id],
         )
 
-        assert db_session.scalars(query).all() == [student_in_assignment]
```

Comment:
In the test I'm keeping the User objects around and just tweaking enough to not get LMSUser. I reckon it's easier to test that the queries work the same way this way.

The assertions are based on h_userid.